frontend/src/metabase/admin/performance/components/PerformanceApp.styled.tsx

```diff
@@ -30,6 +31,15 @@ export const TabsPanel = styled(Tabs.Panel)<{ value: string }>`
   flex-flow: column nowrap;
   flex: 1;
   justify-content: stretch;
-  padding: 1rem 2.5rem;
-  ${props => props.value === PerformanceTabId.Databases && `overflow: hidden;`}
+  ${props =>
+    [
+      PerformanceTabId.Databases,
+      PerformanceTabId.DashboardsAndQuestions,
+    ].includes(props.value as PerformanceTabId) && `overflow: hidden;`}
+`;
+
+export const TabBody = styled(Flex)`
```

Comment:
One thing that's been brought up lately is the preference for CSS modules over styled components. Any chance we can migrate this to a `.css` file?

Reply:
Port'd!